ChangeLog.md

```diff
@@ -37,6 +164,9 @@
 #### AzureRM.EventHub
 * Fixed AlternameName in New-AzureRmEventHubGeoDRConfiguration
 
+#### AzureRM.IotHub
+* Enable tags and Basic Sku to the IotHub
+
```

Comment:
Not sure how IotHub got put down here - can you move this up to the current release?